cocinaborcelle/src/components/CardCompras.tsx

```diff
@@ -0,0 +1,49 @@
+
+import { useState } from 'react';
+import styles from "@/styles/CardCompras.module.css";
+
+type CardComprasProps = {
+    title?: string;
+    number: number;
+}
+
+export default function CardCompras({ title }: CardComprasProps) {
```

Comment:
```suggestion
export default function CardCompra({ compra }: CardCompraProps) {
```

passando compra como prop